pkg/metrics/exporter.go

```diff
@@ -0,0 +1,61 @@
+package metrics
+
+import (
+	"flag"
+	"fmt"
+	"strings"
+	"sync"
+
+	"go.opencensus.io/stats/view"
+)
+
+var (
+	curMetricsExporter view.Exporter
+	metricsMux         sync.RWMutex
+)
+
+var (
+	metricsBackend = flag.String("metricsBackend", "Prometheus", "Backend used for metrics")
```

Comment:
Have you considered exporting to the OpenCensus Agent instead? That would allow your users to choose for themselves if they want Prometheus, Jaeger, Datadog, etc.

Reply:
Thanks, we can add OC Agent as a follow up to limit the scope of this PR

Reply:
Sounds good. Since you already have the exporter behind a flag, you could just add "open-census-agent" as another value that you handle. Alternatively you could just always initialize both exporters and configure them on separate ports.